### PR TITLE
Apple mail imap folder mapping

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-dc49e76d6a/how-to/)

**Comment by catlook** [comment]
Ok, Kelly, at your convenience!

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-c6a246e5ba/how-to/).

**Comment by kmsholcomb** [comment]
Getting to it today.

**Comment by kmsholcomb** [comment]
Hi @catlook. Suggested edits are attached. Please let me know if you have any questions. Thanks!

[Sync IMAP folders for new accounts in Apple Mail_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1303543/Sync.IMAP.folders.for.new.accounts.in.Apple.Mail_editsKH.pdf)
[Sync IMAP folders for new accounts in Apple Mail_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1303544/Sync.IMAP.folders.for.new.accounts.in.Apple.Mail_editsKH.docx)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-d54cb2fea9/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-399b93c876/how-to/).

File: content/rackspace-email/synchronize-imap-folders-for-new-accounts-in-apple-mail.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+type: article
+created_date: '2017-08-23'
+created_by: William Loy
+last_modified_date: '2017-08-23'
+last_modified_by: William Loy
+product: Rackspace Email
+product_url: rackspace-email
+---
+
+This article describes how to synchronize IMAP folders after you set up a new account in Apple Mail. Because Apple Mail does not automatically synchronize IMAP folders, you must complete this process manually through folder mapping. Instructions for several Aple Mail version are included; select the version that applies to you:
```
Comment (stephamon) [inline content/rackspace-email/synchronize-imap-folders-for-new-accounts-in-apple-mail.md]: Oops, should be "Apple Mail" in the last sentence.

Comment (catlook) [reply content/rackspace-email/synchronize-imap-folders-for-new-accounts-in-apple-mail.md]: doh!!
that'll teach me to type and not cut and paste!

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-15a0e263b9/how-to/)

**Comment by stephamon** [comment]
👍

**Pull request merged**